README.md

```diff
@@ -314,6 +314,20 @@ Input parameter - name (string) specifying service name, pod name, replication c
 client.delete_service("redis-service")
 ```
 
+If you want to cascade delete, for example a deployment, you can use the `delete_options` parameter.
+
+```ruby
+deployment_name = 'redis-deployment'
+namespace = 'default'
+delete_options = Kubeclient::Resource.new(
+    apiVersion: 'meta/v1',
+    gracePeriodSeconds: 0,
+    kind: 'DeleteOptions',
+    propagationPolicy: 'Foreground' # Orphan, Foreground, or Background
+)
+client.delete_deployment(deployment_name, namespace, delete_options)
```

Comment:
It's a bit unfortunate `namespace` is a positional parameter in several methods, now adding more params sometimes requires callers to `foo(name, nil, ...)`.
IMHO it's better to add any new params as named params, so this would be:
```
client.delete_deployment(deployment_name, namespace, delete_options: delete_options)
```

[out of scope: it'd nice to eventually also support keyword `namespace:` everywhere.  Would require some careful args parsing so 2nd arg can be either namespace string/nil or an options hash, but would make API clenear.]

Reply:
total agree @cben 
I wanted to change it but a lot of things broke  ;-(